src/core/queryObserver.ts

```diff
@@ -590,10 +590,6 @@ export class QueryObserver<
       return true
     }
 
-    if (result === prevResult) {
-      return false
-    }
```

Comment:
why was this removed please?

Reply:
`shouldNotifyListeners` is only used by `updateResult` that already does: 
```ts
     // Only notify if something has changed
    if (shallowEqualObjects(this.currentResult, prevResult)) {
      return
    }
```
before calling `shouldNotifyListeners`.
That why this condition was not in the code coverage so we can safely remove it or we can keep it and add a test  to directly  call `shouldNotifyListeners` with the appropriate params to cover this condition.

Reply:
right, I think `shouldNotifyListeners` should not make the extra equals check on result and previousResult, so this is fine 👍